src/css/main.less

```diff
@@ -950,6 +950,10 @@ fieldset[disabled] .btn-info.active {
   margin-bottom: 0px;
 }
 
+.add-bottom {
+  margin-bottom: @base-spacing-unit;
```

Comment:
nitpick, but let's use padding-bottom here so that components on the right of the row can use the whole height.